lib/gax.js

```diff
@@ -58,6 +58,9 @@
  *   pageToken.
  * @property {boolean=} isBundling - If set to false and the call is configured
  *   for bundling, bundling is not performed.
+ * @property {number} maxRetries - the maximum number of retrying attempts that
```

Comment:
By private do you mean to not document the property of the object while still keeping the logic?